lib/svgo/plugins.js

```diff
@@ -87,16 +87,16 @@ const createPreset = ({ name, plugins }) => {
         globalOverrides.floatPrecision = floatPrecision;
       }
       if (overrides) {
-        for (const [pluginName, override] of Object.entries(overrides)) {
-          if (override === true) {
+        for (const pluginName of Object.keys(overrides)) {
+          if (!plugins.find(({name}) => name === pluginName)) {
```

Comment:
Better create array with builtin plugin names and use includes

Reply:
Done